docs/test-runners/minitest.md

```diff
@@ -82,6 +82,8 @@ You can remove the second part after we've let you know that the model is suffic
 
 ## Recording test results
 
+Use [minitest-ci](https://github.com/circleci/minitest-ci) to output the test results to a file in minitest. If you already have a CI server storing your test results it may already be installed.
```

Comment:
```suggestion
First, use [minitest-ci](https://github.com/circleci/minitest-ci) to output the test results to a file. If you already store your test results on your CI server, it may already be installed.
```